### PR TITLE
[Week 1] Remove instructions for using Atom

### PR DESCRIPTION
**Pull request by eschwartz**
Week 1 repo still has references to Atom, and instructions for installing it. Got a question from a Gaiman student about it.

I **removed atom.md**. Do we have matching instructions for VSCode, as part of the Week 1 videos? If not, I can updating the Atom docs here for VSCode.

---

**EDIT** Removed the `extended-instructions.md`, and all unused images. The only image that's used is mockup.png:

Screenshot_2022-03-31-17 23 49

### CODE REVIEW

File: extended-instructions.md [hunk 1]
```diff
@@ -29,22 +29,12 @@ Step | Instruction | Screen Shot
...
+To open the *index.html* file in Chrome. 
 
-**NOTE:** There are a few ways to open your assignment in Chrome:
 
-**Option 1: Open from the Finder**
-- Right click on your file in Atom
-- Select Reveal in Finder
-- Double click on the file in your finder window, to open in Chrome
-
-![screencast](./images/open-chrome-from-finder.gif)
```
Comment (eschwartz) [inline extended-instructions.md:41]: this screencast uses Atom. I'm happy giving students fewer options here, and biasing towards the terminal.

**Comment by DoctorHowser** [comment]
Theres this https://github.com/PrimeAcademy/software-installs/blob/main/visual-studio-code-setup.md

Might be worth looking it over, theres a LOT in there.

**Comment by kdszafranski** [comment]
I meant to remove the extended instructions when I changed the assignment for the new videos. 

Thoughts on that? I don't really want to maintain both instructions.

**Comment by kdszafranski** [comment]
> Theres this https://github.com/PrimeAcademy/software-installs/blob/main/visual-studio-code-setup.md
> 
> Might be worth looking it over, theres a LOT in there.

This repo is not linked to regarding just VSCode anywhere I see in the curriculum repo. Students are directed to it for node, postgres, so they may see it then.

**Comment by eschwartz** [comment]
Removed the `extended-instructions.md`, and all unused images. The only image that's used is mockup.png:

Screenshot_2022-03-31-17 23 49

**Pull request merged**